.github/workflows/pr_check.yaml

```diff
@@ -7,12 +7,22 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        cypress-version: [^10, ^11, ^12]
+
+    name: build (Cypress ${{ matrix.cypress-version }})
+
     steps:
       - name: checkout
         uses: actions/checkout@v2
 
       - uses: actions/setup-node@v2
         with:
           node-version: '16'
-      - run: npm install
-      - run: npm test
\ No newline at end of file
+      - name: Install NPM dependencies for SUT
```

Comment:
A bit of nit picking, but I don't think it's correct to add `for SUT` here. Typically `npm install` is just part of the build process, to install whatever dependencies the app needs, as well as all the dev tooling needed.